integrationTesting/testExtensionBuilder.go

```diff
@@ -33,8 +35,21 @@ type BucketFsUploadParams struct {
 	FileSize                 int    `json:"fileSize"`
 }
 
-func (builder *TestExtensionBuilder) WithInstallFunc() {
+func (builder *TestExtensionBuilder) WithFindInstallationsFunc(jsFunctionCode string) *TestExtensionBuilder {
+	builder.findInstallationsFunc = jsFunctionCode
+	return builder
+}
 
+// MockFindInstallationsFunction creates a JS findInstallations function that returns one installation with given JSON array of parameter definitions.
+func MockFindInstallationsFunction(extensionName string, version string, parametersJSON string) string {
+	template := `return [{
+                name: "$NAME$",
+                version: "$VERSION$",
+                instanceParameters: $PARAMS$
+            }]`
+	filledTemplate := strings.Replace(template, "$NAME$", extensionName, 1)
+	filledTemplate = strings.Replace(filledTemplate, "$VERSION$", version, 1)
+	return strings.Replace(filledTemplate, "$PARAMS$", parametersJSON, 1)
```

Comment:
If we need more control over the template, we could use [text/template](https://pkg.go.dev/text/template).